framework/contracts/account/manager/Cargo.toml

```diff
@@ -54,6 +54,7 @@ abstract-adapter = { workspace = true, features = ["test-utils"] }
 abstract-app = { workspace = true, features = ["test-utils"] }
 abstract-testing = { workspace = true }
 abstract-macros = { workspace = true }
+abstract-integration-tests = { workspace = true }
 
```

Comment:
Was thinking what if we mocktest migration against previous abstract version? 
```toml
previous-abstract-manager = { package = "abstract-manager", git = "https://github.com/AbstractSDK/abstract/", tag = "v0.19.2" } #TODO don't forget to update it before after every release
```

Reply:
We don't have to specify old versions for migration tests. We just need to specify an account that has the old version on-chain and load it! So we need to have a clean account that's on the old version that is already instantiated.

Still need to write the logic to retrieve those.

Reply:
Ah, I see, this would allow 2+ versions migration testing as well, nice!